hls4ml/model/optimizer/passes/dense_bn_fuse.py

```diff
@@ -17,7 +17,10 @@ def transform(self, model, node):
         bn_scale = node.weights['scale']
         bn_bias = node.weights['bias']
 
-        fused_weight = bn_scale.data * dense_weight.data
+        if model.config.model_strategy != 'Resource':
```

Comment:
Should be `dense_node.get_attr('strategy')`. Strategy can be defined per model, per layer type and per layer name. So some dense layers can use strategy `Resource` and some `Latency`. Checking it this way can cause issues.